.buildscript/push-javadoc-to-gh-pages.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+echo "Deploying javadoc..."
+
+TDIR=`mktemp -d -t javadoc`
+
+echo "Creating temporary directory..."
+mkdir -v $TDIR/javadoc-latest
+
+echo "Copying files to temporary directory..."
+cp -R build/docs/javadoc $TDIR/javadoc-latest
+
+cd $TDIR
+
+echo "Cloning current gh-pages branch..."
+git clone --quiet --branch=gh-pages https://${GH_TOKEN}@github.com/taskadapter/redmine-java-api gh-pages > /dev/null
```

Comment:
it would be great if the script gave a user-friendly message if GH_TOKEN was not set. this will be a typical mistake in the future...